### PR TITLE
Error handling

### PR DESCRIPTION
**Pull request by pratik-mazumdar-wemotive**

### CODE REVIEW

File: lsp/utils/payload-builder.ts [hunk 1]
```diff
@@ -262,7 +262,7 @@ const confirm = (
       city: initLogistics.context.city,
       country: 'IND',
       timestamp: contextTimestamp,
-      ttl: 'PT30S',
+      ttl: 'PT10S',
```
Comment (wemakshaychavan) [inline lsp/utils/payload-builder.ts:265]: It should be PT30S only

File: lsp/api/v1/services/ondc.service.ts [hunk 2]
```diff
@@ -528,215 +533,211 @@ class OndcService {
...
-            logisticsMessageId = randomUUID();
-            let initRequest = builder.init(payload, lspOnSearchObj, logisticsMessageId, billingContextTimeStamp);
+          console.log('on_search found--->', lspOnSearch);
+          logisticsMessageId = randomUUID();
+          let initRequest = builder.init(payload, lspOnSearchObj, logisticsMessageId, billingContextTimeStamp);
 
-            // wait 2 sec for on_init request to process
-            await new Promise((r) => setTimeout(r, 2000));
+          // wait 0.5 sec for on_init request to process
+          await new Promise((r) => setTimeout(r, 500));
```
Comment (wemakshaychavan) [inline lsp/api/v1/services/ondc.service.ts:549]: what is this?

File: lsp/utils/payload-builder.ts (hunk 1 above)
Comment (pratik-mazumdar-wemotive) [reply lsp/utils/payload-builder.ts:265]: Okay

File: lsp/api/v1/services/ondc.service.ts (hunk 2 above)
Comment (pratik-mazumdar-wemotive) [reply lsp/api/v1/services/ondc.service.ts:549]: on_init timestamp > confirm timestamp therefore added.

File: lsp/utils/payload-builder.ts (hunk 1 above)
Comment (pratik-mazumdar-wemotive) [reply lsp/utils/payload-builder.ts:265]: Adding this into recent MR

**Pull request closed without merging**